src/set_theory/cardinal_ordinal.lean

```diff
@@ -229,6 +230,50 @@ by rw [← aleph_zero, ← aleph_succ, ordinal.succ_zero]
 lemma countable_iff_lt_aleph_one {α : Type*} (s : set α) : countable s ↔ #s < aleph 1 :=
 by rw [← succ_omega, lt_succ, mk_set_le_omega]
 
+private lemma ord_mem {c : cardinal} : c.ord.card.ord = c.ord :=
+by rw card_ord
+
+/-- Ordinals that are cardinals are unbounded. -/
+theorem ord_aleph'_unbounded : unbounded (<) {b : ordinal | b.card.ord = b} :=
+by { rw unbounded_lt_iff, exact λ a, ⟨_, ⟨ord_mem, le_of_lt (lt_ord_succ_card a)⟩⟩ }
+
+theorem eq_aleph'_of_eq_card_ord {o : ordinal} (ho : o.card.ord = o) : ∃ a, (aleph' a).ord = o :=
+⟨cardinal.aleph_idx.rel_iso o.card, by { simp, exact ho }⟩
+
+/-- `ord ∘ aleph'` enumerates the ordinals that are cardinals. -/
+theorem ord_aleph'_eq_enum_card : ord ∘ aleph' = enum_ord ord_aleph'_unbounded :=
+begin
+  rw [←eq_enum_ord, range_eq_iff],
+  exact ⟨aleph'_is_normal.strict_mono, ⟨(λ a, ord_mem), λ b hb, eq_aleph'_of_eq_card_ord hb⟩⟩
+end
+
+/-- Infinite ordinals that are cardinals are unbounded. -/
+theorem ord_aleph_unbounded : unbounded (<) {b : ordinal | b.card.ord = b ∧ ordinal.omega ≤ b} :=
+(unbounded_lt_iff_unbounded_inter_le ordinal.omega).1 ord_aleph'_unbounded
+
+theorem eq_aleph_of_eq_card_ord {o : ordinal} (ho : o.card.ord = o ∧ ordinal.omega ≤ o) :
```

Comment:
Why the conjunctive assumption?

Reply:
Mistake of mine, I've separated the assumption into two.